flake.nix

```diff
@@ -127,13 +127,9 @@
           });
 
         propagatedDeps =
-          [ ((boehmgc.override {
+          [ (boehmgc.override {
               enableLargeConfig = true;
-            }).overrideAttrs(o: {
-              patches = (o.patches or []) ++ [
-                ./boehmgc-coroutine-sp-fallback.diff
```

Comment:
`./boehmgc-coroutine-sp-fallback.diff` is no longer used & can be deleted.

Reply:
What? How? Our `BoehmGCStackAllocator` solution does not work correctly with vanilla boehmgc!